### PR TITLE
CLN: Unify string storage fixture usage

### PR DESCRIPTION
**Pull request by phofl**

### CODE REVIEW

File: pandas/tests/plotting/test_converter.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def test_dont_register_by_default(self):
         call = [sys.executable, "-c", code]
         assert subprocess.check_call(call) == 0
 
-    @td.skip_if_no("matplotlib", min_version="3.1.3")
+    @td.skip_if_no("matplotlib")
```
Comment (topper-123) [inline pandas/tests/plotting/test_converter.py:79]: There is a `td.skip_if_no_mpl` decorator. seems logical to use that?

File: pandas/tests/plotting/test_converter.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def test_matplotlib_formatters(self):
                 assert Timestamp not in units.registry
             assert Timestamp in units.registry
 
-    @td.skip_if_no("matplotlib", min_version="3.1.3")
+    @td.skip_if_no("matplotlib")
```
Comment (topper-123) [inline pandas/tests/plotting/test_converter.py:114]: Same question about `td.skip_if_no_mpl `.

File: pandas/tests/plotting/test_converter.py (hunk 1 above)
Comment (phofl) [reply pandas/tests/plotting/test_converter.py]: I think we can get rid of them altogether, lets see if ci is green

**Review by topper-123 — APPROVED** [review]

**Comment by topper-123** [comment]
Thanks @phofl.

**Pull request merged**